SWIPs/swip-multiple_payment_processing_support.md

```diff
@@ -27,58 +27,177 @@ This SWIP is part of a series of SWIPs (but can be implemented on its own). To s
 
-Storage providers might want to be compensated with a different currency (e.g. an ERC20 token) or they might want to settle their payment using a different blockchain. Furthermore, a storage provider offering other paid services will find appealing not to be forced to support multiple payment systems, but being able to consolidate the payments received under a single technology.
+Furthermore, storage providers might want to be compensated with a different currency or they might want to settle their payment using a different blockchain altogether. 
 
 Finally, new users of Swarm could bootstrap its participation in a payment channel network by providing storage services with zero cost of entry, as described in [Generalised Swap Swear and Swindle games (Tron & Fischer, 2019).](https://www.sharelatex.com/read/yszmsdqyqbvc) 
 
-When it becomes possible for nodes to set their preference for a payment module, developers will be incentivized to implement payment modules on Swarm as it will be easy for users to choose to pay with this module. It will also seamlessly enable multicurrency support and foster interoperation across blockchains without forcing participants to be tied to a single Blockchain or settlement technology.
+When it becomes possible for nodes to set their preference for a payment module, developers will be incentivized to implement payment modules on Swarm as it will be easy for users to choose to pay with this module. It will also seamlessly enable multicurrency support and foster interoperation across blockchains without forcing participants to be tied to a single blockchain or settlement technology.
 
 ## Specification
 <!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for the current Swarm platform and future client implementations.-->
 
-At a high level a payment module is responsible for:
+### Generic payment module
+At a high level a payment module is responsible for sending a payment to a recipient (a peer providing storage services) for services provided. How payments are actually performed will depend on the specific implementation of the payment module. The payment module is then responsible for:
 
-* Accepting an amount (in honey) and a recipient.
-* Resolving the conversion from honey to money (currently by querying the agreed-upon price oracle with the recipient.)
+* Resolving the conversion from honey to money.
 * Ensuring that the user can engage in SWAP accounting for the chosen payment module before payment is due.
```

Comment:
I think this one can be deleted. I think I rewrote this sentence to the one on line 56. Not sure why it is not deleted.

Reply:
You are right, I removed the duplicated item.